### PR TITLE
Discord RPC

### PR DESCRIPTION
**Pull request by dubfib**
I notice your lacking Discord RPC support so I decided why not add it
image
I know the icon isn't the best but atleast it works for now

### CODE REVIEW

**Comment by dubfib** [comment]
Already been tested btw

**Comment by dubfib** [comment]
image
this is on PC on mobile is screwed up

File: scripts/rpc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function RPC(title) {
```
Comment (hmlendea) [inline scripts/rpc.js:1]: Could you rename it to something that contains the name Discord? E.g: DiscordRPC
for clarity

**Review by hmlendea — CHANGES_REQUESTED** [review]

**Comment by hmlendea** [comment]
Is there a way to change the title to Ge**F**orce NOW (capital F).
It looks a bit weird with a lowercase F because right below it it's written in uppercase, and thus is inconsistent.

File: scripts/rpc.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    const client = require('discord-rich-presence')('963128360219869194');
```
Comment (hmlendea) [inline scripts/rpc.js:2]: Will I need to create a new ID instead of 9631...194 ?

**Comment by hmlendea** [comment]
Other than that it looks great and thank you very much for the contribution!

File: scripts/rpc.js (hunk 2 above)
Comment (dubfib) [reply scripts/rpc.js:2]: No, its already created :)

File: scripts/rpc.js (hunk 1 above)
Comment (dubfib) [reply scripts/rpc.js:1]: Sure I can change that right now.

**Comment by dubfib** [comment]
> Is there a way to change the title to Ge**F**orce NOW (capital F). It looks a bit weird with a lowercase F because right below it it's written in uppercase, and thus is inconsistent.

Hmm so the thing is that it might take a bit of time because I need to do something with arguments

**Comment by dubfib** [comment]
I forgot to add something sorry

**Review by hmlendea — APPROVED** [review]

**Comment by hmlendea** [comment]
Looks ok.

I can merge it now if you want.

**Comment by dubfib** [comment]
> Also, a neat idea for a future PR: a toggle to disable this functionality, in case someone does not want it for any reason. _(e.g. a CLI argument like `--disale-discord-rpc`, or a config file)_

I dont know how to do that actually with a command but I can make like a settings.json file

**Comment by hmlendea** [comment]
> > Looks ok.
> > I can merge it now if you want.
> 
> If you want sure but this is only version 1.

As long as it works it's fine.
It can always be further refined later.
But if you plan to do more changes in this PR go ahead, I'm not in a hurry :P

> > Also, a neat idea for a future PR: a toggle to disable this functionality, in case someone does not want it for any reason. _(e.g. a CLI argument like `--disale-discord-rpc`, or a config file)_
> 
> I dont know how to do that actually with a command but I can make like a settings.json file

That would be nice. Probably even better than the CLI argument.
Just make sure to read the file from a location in the home directory, prefferably `~/.config/GeForce NOW` since that directory already exists for this app's cache and electron-specific configs

**Comment by dubfib** [comment]
> > > Looks ok.
> > > I can merge it now if you want.
> > 
> > 
> > If you want sure but this is only version 1.
> 
> As long as it works it's fine. It can always be further refined later. But if you plan to do more changes in this PR go ahead, I'm not in a hurry :P
> 
> > > Also, a neat idea for a future PR: a toggle to disable this functionality, in case someone does not want it for any reason. _(e.g. a CLI argument like `--disale-discord-rpc`, or a config file)_
> > 
> > 
> > I dont know how to do that actually with a command but I can make like a settings.json file
> 
> That would be nice. Probably even better than the CLI argument. Just make sure to read the file from a location within the home directory, preferably `~/.config/GeForce NOW` since that directory already exists for this app's cache and electron-specific configs

I am doing a CLI argument for this actually

**Comment by dubfib** [comment]
image
^^^ when on the home screen/not have a game loaded

image
^^^ when in a game

to disable RPC in the startup add --disable-rpc

**Comment by dubfib** [comment]
The "Not affiliated with NVIDIA" is so then you don't get sued not that sure but just good to have

**Comment by dubfib** [comment]
I already committed and should be ready you can merge now because I don't think I'm going to add more unless I can think... if that happens I'll make another pull request

**Review by hmlendea — APPROVED** [review]

**Pull request merged**